### PR TITLE
Add maxpending which by default is 1000 to control max inflight.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR

### CODE REVIEW

File: searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp [hunk 1]
```diff
@@ -1313,6 +1318,7 @@ App::get_options()
     enum longopts_enum {
         LONGOPT_CLIENT_THREADS,
         LONGOPT_INDEXING_SEQUENCER,
+        LONGOPT_MAX_PENDING,
```
Comment (toregge) [inline searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp:1321]: Usage message is not updated, and order is inconsistent with option array.

**Review by toregge — CHANGES_REQUESTED** [review]

File: searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp:1322]: Fixed

Comment (toregge) [reply searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp:1322]: "documents" is before  "max-pending" in long_opts array, thus LONGOPT_MAX_PENDING should be after LONGOPT_DOCUMENTS in longopts_enum.

**Review by toregge — CHANGES_REQUESTED** [review]

File: searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/apps/vespa-feed-bm/vespa_feed_bm.cpp:1325]: Now they should all be in alphabetical ordering.

**Review by toregge — APPROVED** [review]

**Pull request merged**